internal/cli/clients/client.go

```diff
@@ -695,13 +695,32 @@ func (c *EpinioClient) AppShow(appName string) error {
 	c.ui.Success().
 		WithTable("Key", "Value").
 		WithTableRow("Status", app.Status).
+		WithTableRow("StageId", app.StageID).
 		WithTableRow("Routes", strings.Join(app.Routes, ", ")).
 		WithTableRow("Services", strings.Join(app.BoundServices, ", ")).
 		Msg("Details:")
 
 	return nil
 }
 
+// AppStageID returns the stage id of the named app, in the targeted org
+func (c *EpinioClient) AppStageId(appName string) (string, error) {
```

Comment:
`ID` should be capitalized (https://github.com/golang/go/wiki/CodeReviewComments#initialisms).

Reply:
Right. Will fix.

Reply:
Done with 6e0d35f397412cbd7143acd3450e98600911dd05